ClubMembersSortedCF/src/members/v1/PersonBuilder.java

```diff
@@ -0,0 +1,41 @@
+package members.v1;
+
+import java.time.LocalDate;
+
+public class PersonBuilder {
+    private final Person person = new Person();
+
+    PersonBuilder surname(String surname) {
+        person.setSurname(surname);
+        return this;
+    }
+
+    PersonBuilder firstName(String firstName) {
+        person.setFirstName(firstName);
+        return this;
+    }
+
+    PersonBuilder birthDate(LocalDate birthDate) {
+        person.setDateOfBirth(birthDate);
+        return this;
+    }
+
+    PersonBuilder residence(int zipCode, String settlement, String address) throws InvalidZipCodeException {
+        Residence residence = new Residence(zipCode, settlement, address);
+        person.setResidence(residence);
+        return this;
+    }
+
+    PersonBuilder enterDate(LocalDate enterDate) {
```

Comment:
Small thing, but if I would like to use your Builder, I would generally assume that the name of the method is the same as the name of the field you initialize